librt/libc/include/inttypes.h

```diff
@@ -3,7 +3,7 @@
 #ifndef INTTYPES_H
 #define INTTYPES_H
 
-#include <stdint.h>
+#include "stdint.h"
```

Comment:
Many changes appear to be rebased upon the merged other PR, could you rebase this PR so that is shows less changes?